### PR TITLE
Create Data SQL

### PR DESCRIPTION
**Pull request by IsmailChangezi**
## Description 
- Install PSQL on my Computer.
- Create Database.
- Add tables to the data base 
- Add columns to database
- Perform queries on the items of the table.

### These are the snapshots of the queries.

Screenshot 2023-05-09 172251

### CODE REVIEW

**Review by iambenkis — COMMENTED** [review]
Hi @IsmailChangezi  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that does not have a README file requested for microverse project , Kindly use the template provided by Microverse and customize it  😄 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Gambit142 — COMMENTED** [review]
Hi @IsmailChangezi 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindy remove all placeholder texts from the README.md file. Also kindly remove the Live Demo Section from the README.md file since you would not be deploying the project. 👍 😎 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ CREATE DATABASE vet_clinic;
+\c vet_clinic;
+ CREATE TABLE animals(
+ id INT,
```
Comment (BregornOriginal) [inline schema.sql:4]: [OPTIONAL] Good work creating an animals table 💯 

[OPTIONAL] You could add to the `id` column, the following properties to help you while creating animals. 

`id int NOT NULL IDENTITY(1,1)`

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ name CHAR(50),
+ date_of_birth DATE,
+ escape_attempts INT,
+ neutered BOOLEAN,
+ weight_kg DECIMAL);
```
Comment (BregornOriginal) [inline schema.sql:9]: [OPTIONAL] Good work creating an animals table 💯

[OPTIONAL] You could add to the animal's table, the following properties to help you while creating animals.

`PRIMARY KEY (id)`

File: data.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+\d animals;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-2-3',0,true,10.23 );
+SELECT * FROM animals;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15',2,true,8 ); 
+SELECT * FROM animals;
+UPDATE animals SET neutered = false WHERE id = 1;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-1-7',1,false,15.04 ), (4, 'Devimon', '2017-5-7', 5, true,11);
```
Comment (BregornOriginal) [inline data.sql]: Good work creating animals! 💯 

As there's a lot of information that you need to enter manually, could happen that you have some typos. For example here, the animal `Devimon` has a different date that we have to input. 

- Kindly fix the date_of_birth of Devimon, the correct one is `2017/05/12`
- Kindly remember to use spaces before a `,` (coma) so your code will be more ordered and readable 🥇

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

hand-sketching-stair-slogan-one-step-closer-chalkboard-79948193

## Highlights

✔️ Good work using the template provided 💯 
✔️ Linters are passing
✔️ Good commit history
✔️ Good Pull request description 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 3 above)
Comment (GedeonTS) [inline data.sql]: - As Requested by your last reviewer [here](https://github.com/IsmailChangezi/vet-clinic/pull/1#discussion_r1188639486), Kindly make sure to put the correct birth date for `Devimon`, whose values are supposed to be `(4, 'Devimon', '2017-5-12', '5', 'true', '11');`

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @IsmailChangezi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- Good Summary
- Good PR Description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @IsmailChangezi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**